src/status_im2/contexts/communities/actions/accounts_selection/view.cljs

```diff
@@ -46,9 +52,37 @@
       {:community-name name
        :logo-uri       (get-in images [:thumbnail :uri])}]
      [gesture/scroll-view
-      [rn/view {:style style/content}
+      [:<>
        [quo/text
-        {:style               {:margin-top 24}
+        {:style               style/section-title
+         :accessibility-label :community-rules-title
+         :weight              :semi-bold
+         :size                :paragraph-1}
+        (i18n/label :t/address-to-share)]
+
+       [quo/category
+        {:list-type :settings
+         :data      [{:title             (i18n/label :t/join-as-a-member)
+                      :on-press          #(print "Show All Addresses Sheet")
+                      :description       :text
+                      :action            :arrow
+                      :label             :preview
+                      :label-props       {:type
+                                          :accounts
+                                          :data account-list}
+                      :description-props {:text (i18n/label :t/all-addresses)}}
+                     {:title             (i18n/label :t/for-airdrops)
+                      :on-press          #(print "Show AirDrop Address Sheet")
+                      :description       :text
+                      :action            :arrow
+                      :label             :preview
+                      :label-props       {:type
+                                          :accounts
+                                          :data (subvec account-list 0 1)}
+                      :description-props {:text (:name (account-list 0))}}]}]
```

Comment:
It's dangerous to access vectors like this. At least in Clojure JVM you get index out of bounds if the collection is empty.

Best is `(-> account-list first :name)`